modin/experimental/pandas/__init__.py

```diff
@@ -44,6 +44,7 @@
     read_parquet_glob,
     read_pickle_distributed,
     read_sql,
+    read_xml_glob,
     to_pickle_distributed,
```

Comment:
Do we have an issue to remove the deprecated to_pickle_distributed?

Reply:
https://github.com/modin-project/modin/issues/6856

Reply:
It would be great to deprecate it already in this release.